src/lava/magma/core/run_configs.py

```diff
@@ -70,57 +70,134 @@ def select(self,
 class Loihi1SimCfg(RunConfig):
     """Run configuration selects appropriate ProcessModel -- either
     `SubProcessModel` for a hierarchical Process or else a `PyProcessModel`
-    for a standard Process. Implements and overrides `select()` method of the
-    parent class `RunConfig` to achieve this.
-
-    First, exceptions are checked in `exception_proc_model_map` dictionary
-    as key-value pairs `{Process: ProcessModel}`. These explicit
-    specifications are given precedence over the subsequent logic for
-    `ProcessModel` selection
+    for a standard Process.
 
     If any `SubProcessModel`s are available, then they will be returned first.
 
     In case of `PyProcessModel`s, an appropriate `PyProcessModel` is selected
-    based on the tags associated with it. Tags are set using the `@tag`
-    decorator. If no tags are set for `ProcessModel`s and no tag is selected,
+    based on the tags associated with it (the tags are set using the `@tag`
+    decorator):
+        (a) If no tags are set for `ProcessModel`s and no tag is selected,
     then the first matching `ProcessModel` will be returned.
+        (b) If there is only one ProcessModel for a Process, it will be
```

Comment:
This makes no sense. Imagine there is just a bit-accurate ProcModel but the user explicitly selects bit-approximate then you get something that totally contradicts your selection.

Reply:
If we change this behavior, we also need to change the `Monitor` Process Model, which is currently tagged as `floating_pt`. One of the lava-dnf tutorials is using the `Monitor` Process but selects the tag `fixed_pt`.

Reply:
This is dealt with after second (third?) refactoring.